packages/teams-js/test/public/publicAPIs.spec.ts

```diff
@@ -353,7 +354,7 @@ describe('MicrosoftTeams-publicAPIs', () => {
         Object.keys(expectedContext).forEach((e) => {
           expect(JSON.stringify(expectedContext[e])).toBe(JSON.stringify(context[e]));
         });
-        expect(context.frameContext).toBe(FrameContexts.content);
+        expect(context.frameContext ? context.frameContext : GlobalVars.frameContext).toBe(FrameContexts.content);
```

Comment:
I'm surprised to see this change. Can you explain why it's necessary? When is `context.frameContext` not defined in the test?

Reply:
yeah, good question. This logics exists in transformLegacyContextToAppContext where the old V1 getContext call goes through so I basically translated the same here (without it, test fails complaining content.frameContext undefined).

![image](https://user-images.githubusercontent.com/8062622/214189203-2973c1f9-b7f0-4d33-a1a3-441f4753ee78.png)

Reply:
This context.frameContext transformation should really happen in current getContext() which tests helped to catch(I wasn't sure where to update other than throwing out in test to see). The latest iteration should be up to date.